### PR TITLE
Updates for v3.2.8

### PR DESCRIPTION
**Pull request by lmm**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 05ec2b35c34d6fdd7a16504bfe2a8b0b110549c3

https://deploy-preview-2803--calico.netlify.com

File: _data/versions.yml [hunk 1]
```diff
@@ -905,6 +905,41 @@ v3.3:
...
+      version: v3.2.8
+      url: https://github.com/projectcalico/node/releases/tag/v3.2.8
+    calicoctl:
+      version: v3.2.8
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v3.2.8
+      download_url: https://github.com/projectcalico/calicoctl/releases/download/v3.2.8/calicoctl
+    calico/cni:
+      version: v3.2.8
+      url: https://github.com/projectcalico/cni-plugin/releases/tag/v3.2.8
+      download_calico_url: https://github.com/projectcalico/cni-plugin/releases/download/v3.2.8/calico
```
Comment (tmjd) [inline _data/versions.yml]: These `download_*_url`s are not correct for CNI. They both need `-amd64` appended or we need to add another asset. (The calicoctl one above is correct though.)

File: _data/versions.yml [hunk 2]
```diff
@@ -905,6 +905,41 @@ v3.3:
...
+      download_calico_ipam_url: https://github.com/projectcalico/cni-plugin/releases/download/v3.2.8/calico-ipam
+    calico/kube-controllers:
+      version: v3.2.8
+      url: https://github.com/projectcalico/kube-controllers/releases/tag/v3.2.8
+    networking-calico:
+      version: 3.2.0
+      url: http://git.openstack.org/cgit/openstack/networking-calico/commit/?h=3.2.0
```
Comment (tmjd) [inline _data/versions.yml]: Same as in other PR, this is not a browseable URL.

Comment (lmm) [reply _data/versions.yml]: I'll fix this as per https://github.com/projectcalico/calico/pull/2802/files#r316377866

File: _data/versions.yml (hunk 1 above)
Comment (lmm) [reply _data/versions.yml]: It doesn't appear these URLs are referenced anywhere in v3.x pages or manifests actually. I'll try removing these `download_*_url` fields.

Comment (lmm) [reply _data/versions.yml]: These URLs are used in the integration guide https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/integration#installing-the-calico-cni-plugins

The requirements include amd64 so I think it's okay to not have the arch in the filename.

Comment (tmjd) [reply _data/versions.yml]: There is no asset called `calico` in the release, there is only `calico-amd64`. Right now the wget in "Install the Calico plugins" will not work but it should.
So we either need to copy the `calico-amd64` and upload it as `calico` or update the url to include the `-amd64`.

File: _data/versions.yml (hunk 2 above)
Comment (lmm) [reply _data/versions.yml]: https://github.com/projectcalico/calico/pull/2802/commits/2bd86f89dc074781cfbfba3d50c3230544b50eba

File: _data/versions.yml (hunk 1 above)
Comment (lmm) [reply _data/versions.yml]: Ahh I see what you mean now. dee1aa3540df6b4c8a634fe82409dc44054b0f79

I did a check of the v3.1.x and v3.0.x URLs and those are fine with `calico`. For v3.2.x, the arch started being added to the assets in v3.2.3.

File: _data/versions.yml (hunk 2 above)
Comment (tmjd) [reply _data/versions.yml]: I think this still needs to be updated here. Right?

**Pull request merged**